[Bug] [K8S] Deploying version 3.2.1 with Helm, unable to pull the image.

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

```shell
helm install dolphinscheduler . --set image.tag=3.2.1 
coalesce.go:223: warning: destination for minio.ingress.tls is a table. Ignoring non-table value (false)
Error: INSTALLATION FAILED: failed post-install: timed out waiting for the condition
```
The image repository is incorrect and the image cannot be pulled.
```shell
 Failed to pull image "apache/dolphinscheduler/dolphinscheduler-api:3.2.1": failed to pull and unpack image "docker.io/apache/dolphinscheduler/dolphinscheduler-api:3.2.1": failed to resolve reference "docker.io/apache/dolphinscheduler/dolphinscheduler-api:3.2.1": pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed 
```
```shell
ctr i pull apache/dolphinscheduler/dolphinscheduler-master:3.2.1
apache/dolphinscheduler/dolphinscheduler-master:3.2.1: resolving      |--------------------------------------| 
elapsed: 1.0 s                                         total:   0.0 B (0.0 B/s)                                         
apache/dolphinscheduler/dolphinscheduler-master:3.2.1: resolving      |--------------------------------------| 
elapsed: 1.2 s                                         total:   0.0 B (0.0 B/s)                                         
ctr: failed to resolve reference "apache/dolphinscheduler/dolphinscheduler-master:3.2.1": failed to do request: Head "https://apache/v2/dolphinscheduler/dolphinscheduler-master/manifests/3.2.1": EOF
```
Images cannot be found in the apache/dolphinscheduler repository.

The [values.yaml](https://github.com/apache/dolphinscheduler/blob/dev/deploy/kubernetes/dolphinscheduler/values.yaml) line 34, remove **/dolphinscheduler** can be deployed successfully.
```yaml
  # -- Docker image repository for the DolphinScheduler
  registry: apache
```

### What you expected to happen

deploy successfully.

### How to reproduce

helm install dolphinscheduler . --set image.tag=3.2.1 

### Anything else

_No response_

### Version

3.2.x

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/deploy/kubernetes/dolphinscheduler/README.md
+++ b/deploy/kubernetes/dolphinscheduler/README.md
@@ -174,7 +174,7 @@ Please refer to the [Quick Start in Kubernetes](../../../docs/docs/en/guide/inst
 | image.master | string | `"dolphinscheduler-master"` | master image |
 | image.pullPolicy | string | `"IfNotPresent"` | Image pull policy. Options: Always, Never, IfNotPresent |
 | image.pullSecret | string | `""` | Specify a imagePullSecrets |
-| image.registry | string | `"apache/dolphinscheduler"` | Docker image repository for the DolphinScheduler |
+| image.registry | string | `"apache"` | Docker image repository for the DolphinScheduler |
 | image.tag | string | `"latest"` | Docker image version for the DolphinScheduler |
 | image.tools | string | `"dolphinscheduler-tools"` | tools image |
 | image.worker | string | `"dolphinscheduler-worker"` | worker image |

--- a/deploy/kubernetes/dolphinscheduler/values.yaml
+++ b/deploy/kubernetes/dolphinscheduler/values.yaml
@@ -31,7 +31,7 @@ initImage:
 
 image:
   # -- Docker image repository for the DolphinScheduler
-  registry: apache/dolphinscheduler
+  registry: apache
   # -- Docker image version for the DolphinScheduler
   tag: latest
   # -- Image pull policy. Options: Always, Never, IfNotPresent

--- a/docs/docs/en/guide/installation/kubernetes.md
+++ b/docs/docs/en/guide/installation/kubernetes.md
@@ -14,16 +14,15 @@ If you are a new hand and want to experience DolphinScheduler functions, we reco
 
 ## Install DolphinScheduler
 
-Please download the source code package `apache-dolphinscheduler-<version>-src.tar.gz`, download address: [download address](https://dolphinscheduler.apache.org/en-us/download)
-
-To publish the release name `dolphinscheduler` version, please execute the following commands:
-
-```
-$ tar -zxvf apache-dolphinscheduler-<version>-src.tar.gz
-$ cd apache-dolphinscheduler-<version>-src/deploy/kubernetes/dolphinscheduler
-$ helm repo add bitnami https://charts.bitnami.com/bitnami
-$ helm dependency update .
-$ helm install dolphinscheduler . --set image.tag=<version>
+```bash
+# Choose the corresponding version yourself
+export VERSION=3.2.1
+helm pull oci://registry-1.docker.io/apache/dolphinscheduler-helm --version ${VERSION}
+tar -xvf dolphinscheduler-helm-${VERSION}.tgz
+cd dolphinscheduler-helm
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm dependency update .
+helm install dolphinscheduler .
 ```
 
 To publish the release name `dolphinscheduler` version to `test` namespace:

--- a/docs/docs/zh/guide/installation/kubernetes.md
+++ b/docs/docs/zh/guide/installation/kubernetes.md
@@ -14,16 +14,15 @@ Kubernetes 部署目的是在 Kubernetes 集群中部署 DolphinScheduler 服务
 
 ## 安装 dolphinscheduler
 
-请下载源码包 apache-dolphinscheduler-<version>-src.tar.gz，下载地址: [下载](https://dolphinscheduler.apache.org/zh-cn/download)
-
-发布一个名为 `dolphinscheduler` 的版本(release)，请执行以下命令：
-
-```
-$ tar -zxvf apache-dolphinscheduler-<version>-src.tar.gz
-$ cd apache-dolphinscheduler-<version>-src/deploy/kubernetes/dolphinscheduler
-$ helm repo add bitnami https://charts.bitnami.com/bitnami
-$ helm dependency update .
-$ helm install dolphinscheduler . --set image.tag=<version>
+```bash
+# 自行选择对应的版本
+export VERSION=3.2.1
+helm pull oci://registry-1.docker.io/apache/dolphinscheduler-helm --version ${VERSION}
+tar -xvf dolphinscheduler-helm-${VERSION}.tgz
+cd dolphinscheduler-helm
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm dependency update .
+helm install dolphinscheduler .
 ```
 
 将名为 `dolphinscheduler` 的版本(release) 发布到 `test` 的命名空间中：

